README.md

```diff
@@ -27,14 +27,17 @@ run Pipeline for development:
 ```
 git clone git@github.com:thepoly/pipeline.git
 cd pipeline
-pipenv install
+pipenv install --dev
+brew install postgresql
 npm install
 npx wp --config webpack.development.config.js
 ```
 
 ### Running
 
 ```
+brew services start postgresql
+createdb pipeline //only on first run
```

Comment:
What about moving this up to the Installing section since it only needs to happen once?